ChangeLog

```diff
@@ -14,6 +14,10 @@ What's New in Pylint 2.9.6?
 ===========================
 Release date: TBA
 
+* pyreverse: Show class has-a relationships inferred from the type-hint
```

Comment:
I'd like to put that in 2.10 because it seems like a new feature, what do you think ?

Reply:
Sounds good to me.

Reply:
Fixed while fixing the conflict with the main branch :)